### PR TITLE
Blank text impl

### PR DESCRIPTION
**Pull request by englishman**
Apache Commons `StringUtils#isBlank` OO analog implementation

### CODE REVIEW

**Comment by 0crat** [comment]
@englishman I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/text/BlankText.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+/**
+ * Determines if text is blank (consists of spaces) or no.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Andriy Kryvtsun (kontiky@gmail.com)
+ * @since 0.1
+ */
+public final class BlankText implements Scalar<Boolean> {
```
Comment (dusan-rychnovsky) [inline src/main/java/org/cactoos/text/BlankText.java]: I don't think this is the best naming option. If I see an object of a class `BlankText` I'm going to assume it represents a blank text. This is confusing.

Comment (englishman) [inline src/main/java/org/cactoos/text/BlankText.java]: @dusan-rychnovsky pls, share your opinion here
https://github.com/yegor256/cactoos/issues/6

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@rultor try to merge

**Pull request merged**